generator/templates/client/client.gotmpl

```diff
@@ -32,7 +32,7 @@ type Client struct {
   formats strfmt.Registry
 }
 
-{{ range .Operations }}/*{{ if .Summary }}{{ .Summary }}{{ if .Description }}
+{{ range .Operations }}/* {{ if .Summary }}{{ pascalize .Name }}: {{ .Summary }}{{ if .Description }}
```

Comment:
This is `/*FindPetsByStatus: Finds Pets by status` vs `/* FindPetsByStatus: Finds Pets by status`.